### PR TITLE
Add relval wf `139.005` to test the AlcaEcalPhiSym workflow for Run3

### PR DESCRIPTION
**Pull request by francescobrivio**
#### PR description:
This PR builds on top of #36988 by adding a relval wf to the matrix in order to test the new ECAL AlCaPhiSym workflow for Run3. It produces nano as output and it will be eventually tested in a tier0 replay (we still need to sort out some details).

#### PR validation:
Tested with:
`runTheMatrix.py -l 139.005 --ibeos -j8 -t8`

#### Backport:
Not a backport. A Backport to 12_3 will be opened soon.

FYI @simonepigazzini @rappoccio @cms-sw/alca-l2

### CODE REVIEW

**Comment by francescobrivio** [comment]
test parameters:

 - workflows=139.005

**Comment by francescobrivio** [comment]
test parameters:

 - workflows=139.005

**Comment by tvami** [comment]
@cmsbuild , ping

**Comment by francescobrivio** [comment]
ping bot

**Comment by smuzaffar** [comment]
there was github webhook issues that is why bot did not react to this message. Things look better now and bot has already started the code-checks

**Comment by tvami** [comment]
@smuzaffar it actually finished with the code checks based on this https://cmssdt.cern.ch/jenkins/job/run-pr-code-checks/28995/consoleFull but it never changed the state here
Screen Shot 2022-03-25 at 11 17 11

**Comment by smuzaffar** [comment]
huum, may be bot was not able to add the comment, I am restarted the code-checks job

**Comment by francescobrivio** [comment]
@cmsbuild please test

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -506,6 +503,8 @@
 steps['RunZeroBias2021']={'INPUT':InputInfo(dataSet='/ZeroBias/Commissioning2021-v1/RAW',label='2021Commissioning',events=100000,location='STD', ls=Run2021Commissioning)}
 steps['RunHLTPhy2021']={'INPUT':InputInfo(dataSet='/HLTPhysics/Commissioning2021-v1/RAW',label='2021Commissioning',events=100000,location='STD', ls=Run2021Commissioning)}
 steps['RunNoBPTX2021']={'INPUT':InputInfo(dataSet='/NoBPTX/Commissioning2021-v1/RAW',label='2021Commissioning',events=100000,location='STD', ls=Run2021Commissioning)}
+steps['AlCaPhiSym2021']={'INPUT':InputInfo(dataSet='/AlCaPhiSym/Commissioning2021-v1/RAW',label='2021Commissioning',run=[346512],location='STD')}
+steps['AlCaLumiPixels2021']={'INPUT':InputInfo(dataSet='/AlCaLumiPixelsCountsExpress/Commissioning2021-v1/RAW',label='2021',run=[346512],events=100000,location='STD')}
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_steps.py:507]: Why do you choose `label='2021'` instead of `label='2021Commissioning`' here?

Comment (francescobrivio) [reply Configuration/PyReleaseValidation/python/relval_steps.py:507]: Hi @srimanob!
I didn't wrote this line actually, I just moved it down here from above to be together with the other 2021 datasets. Personally I don't have any preference on the label, do you prefer I change it to '2021Commissioning'?

Comment (srimanob) [reply Configuration/PyReleaseValidation/python/relval_steps.py:507]: Hi @francescobrivio 
Ah, yeah. the line was somehow put above 2021 commissioning before. Would you mind to fix its label? Thanks.

Comment (francescobrivio) [reply Configuration/PyReleaseValidation/python/relval_steps.py]: Done in 6e9efee

**Comment by francescobrivio** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
+Upgrade

**Pull request merged**